### PR TITLE
cpu/nrf9160: add periph_flashpage support

### PR DESCRIPTION
**Pull request by dylad**
### Contribution description

This PR adds periph_flashpage support for nRF9160 family. Changes in nrf5x_common driver is minimal (only the erase page is different from nRF51/nRF52).

### Testing procedure

Flash tests/periph_flashpage on `nrf9160dk` board and play with the test app.

I provide the following [gist](https://gist.github.com/dylad/8285886a66a393773f803a497c4bbcbe) for some test output.
I also tried riotboot_serial on my custom nrf9160 board and it works like a charm.

### Issues/PRs references
None.

### CODE REVIEW

File: cpu/nrf9160/include/cpu_conf.h [hunk 1]
```diff
@@ -42,6 +42,21 @@ extern "C" {
 #define CPU_IRQ_NUMOF                   (65U)   /**< nRF9160 specific IRQ count */
 /** @} */
 
+/**
+ * @brief   Flash page configuration
```
Comment (aabadie) [inline cpu/nrf9160/include/cpu_conf.h:46]:
```suggestion
 * @name    Flash page configuration
```

Comment (dylad) [reply cpu/nrf9160/include/cpu_conf.h]: Fixed.

**Review by benpicco — APPROVED** [review]
LGTM

**Comment by dylad** [comment]
Can I squash ?

**Comment by dylad** [comment]
Squashed.

**Comment by dylad** [comment]
Added another commit for Kconfig which I have completely forgotten.

**Comment by dylad** [comment]
@benpicco @aabadie 
Care to take another look please ? Just add Kconfig after the ACK so it shouldn't be a big deal.

**Comment by dylad** [comment]
thanks @benpicco !

**Pull request merged**